onur/styles.css

```diff
@@ -0,0 +1,241 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    color:#18181a;
+}
+html {
+    font-family: "Open Sans", sans-serif;
+}
+/* navbar section */
+a {
+    text-decoration: none;
+    color: #363636;
+    font-size: 1rem;
+}
+.navbar ul li a:hover {
+    text-decoration:underline;
+    color: black;
+}
+
+.text-center {
+    text-align: center;
+}
+.heading-small {
+    font-size: 1.2rem;
+    font-weight: 400;
+    text-align: left;
+}
+
+.btn {
+    color:rgb(30, 143, 218);
+    border: 1px solid rgb(30, 143, 218);
+    border-radius: 5px;
+    padding: 0.6rem;    
+}
+.btn:hover {
+    background-color: rgb(30, 143, 218);
+    color:white;
+}
+.btn:active {
+    box-shadow: 0 0 2px 2px #3e8fc5;
+}
+.conteiner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.25rem;
+    background: rgb(255, 255, 255);
+    height: 70px;
+    border-bottom: 1px solid rgb(222 226 230);
+    box-shadow: rgba(0, 0, 0, 0.05) 0px 7px 11px 0px;  
+}
+.navbar ul {
+    display: flex;
+    align-items: center;
+    gap: 1.2rem; 
+    list-style-type: none;
+}
+/* description section */
+.description {
+    background: (255, 255, 255);
+    max-width: 700px;
+    width: 70%;
+    height: 200px;
```

Comment:
I'd avoid setting height of this element explicitly as it may break on a different kind device. Instead, add enough spacing around the text (use margin or padding, which one is the most appropriate) and let the content take as much space as it needs to.